app/models/manageiq/providers/storage_manager/cinder_manager/refresh_helper_methods.rb

```diff
@@ -0,0 +1,173 @@
+module ManageIQ::Providers::StorageManager::CinderManager::RefreshHelperMethods
```

Comment:
@hsong-rh Actually, this is contrary to the existing pattern. All of the other parsers have all the parsing code in the same file. As I said, these aren't "helper methods" this is parsing code.